en/editor/publish/native-options.md

```diff
@@ -1,195 +1,218 @@
-> 2. After the project is imported with Android Studio, it is an independent existence and does not depend on the **Build** panel. If you need to modify the ABI, you can directly modify the `PROP_APP_ABI` property in `gradle.properties` file as shown below:
+> **Note**:
 >
->     ![modify abi](publish-native/modify_abi.png)
-
-#### Use Debug Keystore
+> 1. After you select an ABI to build, build another ABI without cleaning. At this time, the so of both ABIs will be packaged into the apk. This is the default behavior of Android Studio. If you use Android Studio to import the project, after selecting an ABI to build, first execute **Build -> Clean Project** and then build another ABI. At this time, only the latter ABI will be packaged into the apk.
+>
+> 2. After the project is imported with Android Studio, it is an independent existence and does not depend on the build release panel. If you need to modify the ABI, just modify the **PROP_APP_ABI** property in the **gradle.properties** file.
+>
+> ![modify abi](publish-native/modify_abi.png)
 
-Android requires that all APKs be digitally signed with a certificate before they can be installed. A default keystore is provided, check the **Use Debug Keystore** to use the `default keystore`. If you need to customize the keystore, you can remove the **Use Debug Keystore** checkbox. Please refer to the official [Android Documentation](https://developer.android.com/studio/publish/app-signing) for details.
+#### Use the debug keystore
 
-Android requires that all APKs must be digitally signed with a certificate before they can be installed. Cocos Creator provides a default keystore by checking **Use Debug Keystore** to use it. If you need to customize the keystore, you can remove the **Use Debug Keystore** checkbox, refer to the [Official Documentation](https://developer.android.com/studio/publish/app-signing) for details.
+Android requires that all APKs must be digitally signed with a certificate before they can be installed. Cocos Creator provides a default keystore. Check **Use debug keystore** to use the default keystore. If the developer needs to customize the keystore, uncheck the **Use debugging keystore** check. For details, please refer to the [Official Documentation](https://developer.android.google.cn/studio/publish/app-signing).
 
-#### Screen Orientation
+#### Screen orientation
 
-The screen orientation currently includes **Portrait**, **Landscape Left** and **Landscape Right**.
+The screen orientation currently includes **Portrait**, **Landscape Left**, and **Landscape Right**.
 
-- **Portrait**: the screen is placed vertically with the **Home** button on the bottom.
-- **Landscape Left**: the screen is placed horizontally, with the **Home** button on the left side of the screen.
-- **Landscape Right**: the screen is placed horizontally, with the **Home** button on the right side of the screen.
+- **Portrait**: The screen is upright and the Home button is down.
```

Comment:
I remember you said that the first letter after the colon should be lowercase, right?

Reply:
yes. fixed.